queue.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+
+class QueueItem(object):
+    def __init__(self, data, prev_item=None, next_item=None):
+        self.data = data
+        self.next_item = next_item
+        self.prev_item = prev_item
+
+    def __str__(self):
+        return str(self.data)
+
+
+class Queue(object):
+    def __init__(self, first_item=None, last_item=None):
+        self.first_item = first_item
```

Comment:
When I read your init function, I don't know which end is which. If only there was a way to add documentation to your function...